src/50_Samples_%26_Templates/Quiz.html

```diff
@@ -0,0 +1,164 @@
+<!---{
+"preview": "default",
+"experiments": ["amp-bind"]
+}--->
+<!--
+#### Introduction
+
+This is a sample template for a quiz in AMP. There are 4 questions in total.
```

Comment:
suggestion: 'This is a sample template for a quiz in AMP consisting of four questions.'